zlib-rs/src/allocate.rs

```diff
@@ -442,6 +445,14 @@ mod tests {
         unaligned_allocator_help::<Align64>()
     }
 
+    #[test]
+    fn align_size() {
+        // posix_memalign requires that the alignment be both a power of two and
+        // a multiple of sizeof(void*).
+        assert_eq!(ALIGN.count_ones(), 1);
+        assert_eq!(ALIGN as usize % mem::size_of::<*mut c_void>(), 0);
+    }
+
```

Comment:
you can write this as 

    const _: () = assert!(ALIGN.count_ones() == 1);
    const _: () = assert!(ALIGN as usize % mem::size_of::<*mut c_void>() == 0);

then they are evaluated at compile time